catppuccin_tmux.conf

```diff
@@ -130,21 +130,31 @@ set -wgF pane-border-style "#{E:@catppuccin_pane_border_style}"
 
 %if "#{==:#{@catppuccin_window_status_style},basic}"
 
-  set -gq @catppuccin_window_left_separator "█"
-  set -gq @catppuccin_window_middle_separator "█"
-  set -gq @catppuccin_window_right_separator "█ "
+  set -gq @catppuccin_window_left_separator " "
+  set -gq @catppuccin_window_middle_separator " "
+  set -gq @catppuccin_window_right_separator " "
 
 %elif "#{==:#{@catppuccin_window_status_style},rounded}"
 
-  set -gq @catppuccin_window_left_separator ""
-  set -gq @catppuccin_window_middle_separator "█"
-  set -gq @catppuccin_window_right_separator " "
+  set -gq @catppuccin_window_left_separator "#[fg=#{@_ctp_status_bg},reverse]#[none]"
+  set -gq @catppuccin_window_middle_separator " "
+  set -gq @catppuccin_window_right_separator "#[fg=#{@_ctp_status_bg},reverse]#[none]"
```

Comment:
Not a fan of placing color attributes in this option, I guess it makes the window logic a lot simpler?

Reply:
Why aren't you a fan? It makes the window logic incredibly simpler